README.md

```diff
@@ -11,3 +9,59 @@ Make sure that you have the compatible versions of Visual Studio with .NET MAUI
+
+### Resource view
+Group appointments based on the resources associated with them in the timeline day, timeline week, timeline workweek, and timeline month views.
+
+### Right to left (RTL)
+The .NET MAUI Scheduler supports displaying the Scheduler component from right to left for users working with RTL languages like Hebrew and Arabic.
+
+## Related Links
+
+[Learn More about .NET MAUI Scheduler](https://www.syncfusion.com/maui-controls/maui-scheduler?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace)<br/><br/>
+
+[Download Free Trial](https://www.syncfusion.com/downloads/maui/confirm?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace) <br/><br/>
+
+[Pricing](https://www.syncfusion.com/sales/teamlicense?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace) <br/><br/>
+
+[Documentation](https://help.syncfusion.com/maui/scheduler/getting-started?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace) <br/><br/>
+
+[View Demos](https://github.com/SyncfusionExamples/maui-scheduler-examples?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace) <br/><br/>
+
+[Community Forums](https://www.syncfusion.com/forums/maui?control=sfscheduler?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace)<br/><br/>
+
+[Suggest a Feature](https://www.syncfusion.com/feedback/maui?control=sfscheduler?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace)
+
+ 
+
+## About Syncfusion .NET MAUI Controls
+
+Syncfusion's [.NET MAUI UI controls](https://www.syncfusion.com/maui-controls/maui-scheduler?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace) library is the only suite that you will ever need to build an application since it contains over 70 high-performance, lightweight, modular, and responsive UI controls in a single package. In addition to Scheduler, we provide popular MAUI Controls such as [DataGrid](https://www.syncfusion.com/maui-controls/maui-datagrid?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace), [Charts](https://www.syncfusion.com/maui-controls/maui-cartesian-charts?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace), [Scheduler](https://www.syncfusion.com/maui-controls/maui-scheduler?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace), [ListView](https://www.syncfusion.com/maui-controls/maui-listview?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace), and [Excel Library](https://www.syncfusion.com/document-processing/excel-framework/maui?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-scheduler-trial-vs_marketplace).
```

Comment:
Controls, over 80, since you are working on Scheduler no need to add the below underlined content:
![image](https://github.com/SyncfusionExamples/getting-started-with-dotnet-maui-scheduler/assets/95272441/81625dc2-531b-4506-bc85-abc3b97821a7)

Reply:
Updated